flow/core/kernel/vehicle/traci.py

```diff
@@ -87,6 +87,8 @@ def __init__(self,
 
         # old speeds used to compute accelerations
         self.previous_speeds = {}
+        # The time that previous speed is recorded, used to calculate realized_accel
+        self.previous_time = 0
```

Comment:
This is not actually used. Sorry, I think I just forgot to delete it after I later realized I don't need it.